clouddriver-core/src/main/groovy/com/netflix/spinnaker/clouddriver/orchestration/AtomicOperations.groovy

```diff
@@ -57,4 +57,8 @@ final class AtomicOperations {
   public static final String RUN_JOB = "runJob"
   public static final String DESTROY_JOB = "destroyJob"
   public static final String CLONE_JOB = "cloneJob"
+
+  // Serialize operations
+  public static final String SERIALIZE_APPLICATION = "serializeApplication"
+  public static final String DESERIALIZE_APPLICATION = "deserializeApplication"
```

Comment:
If there is no `DESERIALIZE_APPLICATION` operation yet, we don't need the constant.